Source/Validation.swift

```diff
@@ -218,9 +218,9 @@ extension DataRequest {
 // MARK: -
 
 extension DownloadRequest {
-    /// A closure used to validate a request that takes a URL request, a URL response and destination URL, and returns
-    /// whether the request was valid.
-    public typealias Validation = (URLRequest?, HTTPURLResponse, URL?) -> ValidationResult
+    /// A closure used to validate a request that takes a URL request, a URL response, a temporary URL and a 
+    /// destination URL, and returns whether the request was valid.
+    public typealias Validation = (URLRequest?, HTTPURLResponse, _ temporary: URL?, _ destination: URL?) -> ValidationResult
```

Comment:
You've already broken the column limit, why not go all the way and label all of the parameters? Labeling only two seems odd.